WordPress/Classes/ViewRelated/Reader/WPStyleGuide+Reader.swift

```diff
@@ -56,40 +56,64 @@ extension WPStyleGuide {
     // MARK: - Card Attributed Text Attributes
 
     @objc public class func readerCrossPostTitleAttributes() -> [NSAttributedString.Key: Any] {
-        let font = WPStyleGuide.serifFontForTextStyle(Cards.crossPostTitleTextStyle)
+        if FeatureFlag.readerImprovements.enabled {
+            let font = UIFont.preferredFont(forTextStyle: .subheadline).semibold()
+            return [
+                .font: font,
+                .foregroundColor: UIColor.label
+            ]
+        } else {
+            let font = WPStyleGuide.serifFontForTextStyle(Cards.crossPostTitleTextStyle)
 
-        let paragraphStyle = NSMutableParagraphStyle()
-        paragraphStyle.lineSpacing = Cards.crossPostLineSpacing
+            let paragraphStyle = NSMutableParagraphStyle()
+            paragraphStyle.lineSpacing = Cards.crossPostLineSpacing
 
-        return [
-            .paragraphStyle: paragraphStyle,
-            .font: font,
-            .foregroundColor: UIColor.text
-        ]
+            return [
+                .paragraphStyle: paragraphStyle,
+                .font: font,
+                .foregroundColor: UIColor.text
+            ]
+        }
     }
 
     @objc public class func readerCrossPostBoldSubtitleAttributes() -> [NSAttributedString.Key: Any] {
-        let font = WPStyleGuide.fontForTextStyle(Cards.crossPostSubtitleTextStyle, symbolicTraits: .traitBold)
-
-        let paragraphStyle = NSMutableParagraphStyle()
-        return [
-            .paragraphStyle: paragraphStyle,
-            .font: font,
-            .foregroundColor: UIColor(light: .gray(.shade40), dark: .systemGray)
-        ]
+        if FeatureFlag.readerImprovements.enabled {
+            let font = UIFont.preferredFont(forTextStyle: .footnote).semibold()
```

Comment:
Just a minor thing: I usually use `WPStyleGuide.fontForTextStyle(_:fontWeight:)` since I noticed that there are some code that deals with font scaling issues for certain weights. But that code has been there for a long while and I'm not sure if it's still relevant. 😅

Reply:
Based off the comment in that function:
```swift 
/// WORKAROUND: Some font weights scale up well initially but they don't scale up well if dynamic type
///     is changed in real time.  Creating a scaled font offers an alternative solution that works well
///     even in real time.
```

It appears that the code was for dynamically scaling bolded font when changed in real time. I did a quick check on this cell and don't see any issues with changing the font size. So you may be right and it might no longer be needed.